### ISSUE TITLE
Add health check to foreign wallet API

### ISSUE THREAD

**Issue by cayblood** [mimblewimble/grin#2619]
The foreign wallet API seems only to have two POST apis. I want to host my foreign wallet API behind an AWS load balancer so that it can do the SSL termination for me. However, it requires some simple GET request to check whether the listener is up. It would be nice to have some simple status call in addition to the two POST commands.

**Comment by yeastplume** [comment]
Yes, and it should also be returning the wallet version. Can you open this in the grin-wallet repo though?
